shared/chat/conversation/list-area/index.desktop.tsx

```diff
@@ -744,16 +786,7 @@ const OrdinalWaypointInner = (p: OrdinalWaypointProps) => {
   } else {
     return content
   }
-}
-
-const OrdinalWaypoint = React.memo<OrdinalWaypointProps>(
-  OrdinalWaypointInner,
-
-  (prevProps, nextProps) =>
```

Comment:
These custom memo function are footguns i think, better to just memo the props further upstream. here we didn't capture if previous changed (or others) which would cause this to memoize incorrectly